src/components/Addbook.jsx

```diff
@@ -73,15 +73,15 @@ const Div = styled.div`
     .title {
       height: 2.813rem;
       border-radius: 4px;
-      border: solid 1px #2e90f0;
+      border: solid 1px #e1e1e1;
       background-color: #fff;
       padding-left: 1rem;
       width: 55.15%;
     }
     .author {
       height: 2.813rem;
       border-radius: 4px;
-      border: solid 1px #2e90f0;
+      border: solid 1px #e1e1e1;
```

Comment:
- your nav links should be correctly aligned to the centre with the header and user icon as seen below 👍🏾 
- the colour of the header which should be blue Is different from the original design please fix this error 👍🏾 
- the user icon should have some significant padding to the right to proper resemble the original design 

required design 
![image](https://github.com/shihabrafio/BookStore/assets/60612329/2f8c43ee-ff1d-4d69-8168-1c5557a6c0b9)

your design 
![image](https://github.com/shihabrafio/BookStore/assets/60612329/41b9f7db-1aa8-48ce-9aea-8d961b2460c1)